src/connection/sql_connection.go

```diff
@@ -41,6 +41,7 @@ func (sc SQLConnection) Close() {
 
 // Query runs a query and loads results into v
 func (sc SQLConnection) Query(v interface{}, query string) error {
+	log.Debug("Running query: %s", query)
```

Comment:
Just to add some more insights for the future, would make sense to compute the time spent to execute a query? 
and just log it after?

Reply:
I considered it, but I was a bit concerned about computing it regardless of the log level (we don't have an easy way to check it at the moment). Do you think it is worth it?

Reply:
I forgot the log from our SDK do not allow getting the level 😕  

we can always add it in the future if needed.